code/modules/power/cell.dm

```diff
@@ -426,14 +426,17 @@
 	desc = "A rechargeable starch based power cell."
 	icon = 'icons/obj/service/hydroponics/harvest.dmi'
 	icon_state = "potato"
-	charge = 100
 	maxcharge = STANDARD_CELL_CHARGE * 0.3
 	charge_light_type = null
 	connector_type = null
 	custom_materials = null
 	grown_battery = TRUE //it has the overlays for wires
 	custom_premium_price = PAYCHECK_CREW
 
+/obj/item/stock_parts/cell/potato/Initialize(mapload, override_maxcharge)
+	charge = maxcharge * 0.03
```

Comment:
```suggestion
	charge = maxcharge * 0.3
```

I think this is correct since STANDARD_CELL_CHARGE was 1000 before the energy changes.